bplan/management/commands/load_bplan.py

```diff
@@ -230,7 +246,8 @@ def handle(self, *args, **options):
             if not geometry.valid:
                 invalid_items.append(planname)
 
-            point = self._calculate_point(multipolygon)
+            point = self._calculate_point(multipolygon, points)
+            points.append(point)
```

Comment:
I know this is data duplication. But for some reason, I couldn't do `Bplan.objects.all()` inside the script. :(